execution/gethexec/sequencer.go

```diff
@@ -88,34 +88,6 @@ type SequencerConfig struct {
 	EnableCaffNode bool `koanf:"enable-caff-node"`
 }
 
-type DangerousConfig struct {
-	Timeboost TimeboostConfig `koanf:"timeboost"`
-}
-
-type TimeboostConfig struct {
```

Comment:
why are we deleting this again lol

Reply:
So the reason is that the merge doesn't happen properly and it duplicates these functions. I am deleting the v3.5.4 version and keeping the v3.5.6 version.